.github/workflows/irc-notifications.yaml

```diff
@@ -1,5 +1,5 @@
 name: "Push Notification"
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
maybe `push` should only do a very simple test suite run?

Reply:
I think if we do it on pull_request we dont need it on push. but i can restore that.